### PR TITLE
Fixes #7495 - Create/delete autosign entries via API

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

**Comment by dLobatog** [comment]
Thanks @xprazak2  - please add the appropriate permissions for tests to pass, PR looks good.

**Comment by xprazak2** [comment]
I added the permissions, hopefully it turns 🍏

**Comment by dLobatog** [comment]
Thanks @xprazak2 !

File: app/controllers/api/v2/autosign_controller.rb [hunk 1]
```diff
@@ -19,6 +39,10 @@ def setup_proxy
       def allowed_nested_id
         %w(smart_proxy_id)
       end
+
+      def handle_proxy_error exception
+        render :status => 500, :json => { "error": exception.message }
```
Comment (mmoll) [inline app/controllers/api/v2/autosign_controller.rb:44]: this seems to break on Ruby 2.0 and 2.1, i.e. http://ci.theforeman.org/job/packaging_build_deb_coreproject/arch=x86,label=debian,os=jessie/4129/console
I guess it should be rather `{ :error => exception.message }`

**Pull request merged**